techniques/css/C42.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 	<head>
-		<title>Using min-height and min-width to ensure sufficient target spacing</title>
+		<title>Using CSS to ensure sufficient target spacing</title>
```

Comment:
If this example is now just for target size (enhanced), which just talks about size and not spacing, the title and h1 should be adjusted accordingly

Reply:
[Already made that change](https://github.com/w3c/wcag/pull/3231/files)—for some reason, it's tricky to find though.